solutions/landing-zone-v2/onboarding-tenant.md

```diff
@@ -1,23 +1,23 @@
-# Tenant Onboarding
```

Comment:
rename this file to become onboarding-client.md

Reply:
Done.